lib/trello/board.rb

```diff
@@ -72,7 +72,8 @@ def update!
       fail "Cannot save new instance." unless self.id
 
       @previously_changed = changes
-      @changed_attributes.clear
+      @changed_attributes.try(:clear)
+      changes_applied if respond_to?(:changes_applied)
```

Comment:
@wokenqingtian I wonder if it would be worth adding a private method to `BasicData`:

```rb
def clear_changes
  @changed_attributes.clear if @changed_attributes.respond_to?(:clear)
  changes_applied if respond_to?(:changes_applied)
end
```